docs/tutorial/node/src/creating_the_tiles_from_js.md

```diff
@@ -0,0 +1,19 @@
+# Creating The Tiles From JavaScript
+
+What we'll do is take the list of tiles declared in the .slint language, duplicate it, and shuffle it.
+We'll do so by accessing the `memory_tiles` property through the JavaScript code - in our case `memory_tiles`.
+Since `memory_tiles` is an array in the `.slint` language, it's represented as a JavaScript [`Array`](https://slint-ui.com/releases/0.3.5/docs/node/).
+We can't modify the model generated by the .slint, but we can extract the tiles from it, and put it
+in a [`slint.ArrayModel`](https://slint-ui.com/releases/0.3.5/docs/node/classes/arraymodel.html) which implements `Model`.
```

Comment:
```suggestion
in a [`slint.ArrayModel`](https://slint-ui.com/releases/0.3.5/docs/node/classes/arraymodel.html) which implements the [`Model`](https://slint-ui.com/releases/0.3.5/docs/node/interfaces/model.html) interface.
```

That said, we need to make sure that those links remain up-to-date. I suggest to do what the other tutorials do and use `https://slint-ui.com/docs/node` as base url, not `https://slint-ui.com/releases/0.3.5/docs/node`.